### PR TITLE
Update component tests

### PR DESCRIPTION
**Pull request by billingsley-john**
## Proposed changes

PR which refactors the parametric component tests to make use of global components. Additional points calculation tests and default parameter tests have been added.
This PR also updates the stp/stl filenames of some components - however, checks need to be performed to ensure this doesn't break any neutronics examples.

## Types of changes

What types of changes does your code introduce to the Paramak?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Code refactoring
- [x] Documentation Update (if none of the other choices apply)
- [x] New tests

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Pep8 applied
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

No further comments

### CODE REVIEW

**Comment by billingsley-john** [comment]
Code coverage has decreased as some error raising tests have been removed/refactored. Thanks @RemDelaporteMathurin for pointing this out.

File: paramak/parametric_components/blanket_constant_thickness_arc_v.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def __init__(
         thickness,
         stp_filename="BlanketConstantThicknessArcV.stp",
         stl_filename="BlanketConstantThicknessArcV.stl",
-        material_tag="blanket_material",
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_components/blanket_constant_thickness_arc_v.py:33]: Docstrings should be changed accordingly :-)

File: paramak/parametric_components/center_column_circular.py [hunk 2]
```diff
@@ -30,7 +30,7 @@ def __init__(
         outer_radius,
         stp_filename="CenterColumnShieldCircular.stp",
         stl_filename="CenterColumnShieldCircular.stl",
-        name="center_column",
+        name="center_column_shield",
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_components/center_column_circular.py:33]: Docstrings should be changed accordingly :-)

File: paramak/parametric_components/poloidal_field_coil_case_set.py [hunk 3]
```diff
@@ -30,10 +30,10 @@ def __init__(
...
         casing_thicknesses,
         center_points,
-        stp_filename="PoloidalFieldCoil.stp",
-        stl_filename="PoloidalFieldCoil.stl",
-        name="pf_coil",
-        material_tag="pf_coil_mat",
+        stp_filename="PoloidalFieldCoilCaseSet.stp",
+        stl_filename="PoloidalFieldCoilCaseSet.stl",
+        name="pf_coil_case_set",
+        material_tag="pf_coil_case_mat",
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_components/poloidal_field_coil_case_set.py:36]: Docstrings should be changed accordingly :-)

File: paramak/parametric_components/poloidal_field_coil_case_set_fc.py [hunk 4]
```diff
@@ -14,8 +14,8 @@ class PoloidalFieldCoilCaseSetFC(RotateStraightShape):
             coil casing (cm). If float then the same thickness is applied to
             all coils. If list of floats then each entry is applied to a
             seperate pf_coil, one entry for each pf_coil.
-        stp_filename (str, optional): defaults to "PoloidalFieldCoil.stp".
-        stl_filename (str, optional): defaults to "PoloidalFieldCoil.stl".
+        stp_filename (str, optional): defaults to "PoloidalFieldCoilCaseSetFC.stp".
+        stl_filename (str, optional): defaults to "PoloidalFieldCoilCaseSetFC.stl".
         name (str, optional): defaults to "pf_coil".
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_components/poloidal_field_coil_case_set_fc.py:19]:
```suggestion
        name (str, optional): defaults to "pf_coil_case_set_fc".
```

**Review by RemDelaporteMathurin — COMMENTED** [review]
Minor docstrings stuff, feel free to ignore

**Comment by billingsley-john** [comment]
Codecov is indicating that the height check for center_column_shield_cylinder.py is not being performed in the tests, despite this test passing locally.

**Comment by RemDelaporteMathurin** [comment]
> Codecov is indicating that the height check for center_column_shield_cylinder.py is not being performed in the tests, despite this test passing locally.

The reason why the tests are passing is because it catches the "inner radius bigger than outer radius" error (cause it's still respectively 40 and 20).

To catch the height as None, those have to be reset to original (20 and 40).

**Comment by RemDelaporteMathurin** [comment]
@billingsley-john I couldn't find a way of suggesting this code in the review. Instead I pushed a commit. Hope it's ok.

File: tests/test_parametric_components/test_BlanketCutterParallels.py [hunk 5]
```diff
@@ -5,48 +5,48 @@
...
-        test_shape.solid
-
-        assert test_shape.solid is not None
-
-        test_shape.cut = [cut_shape]
-        test_shape.solid
-
+        self.test_shape.cut = cut_shape
+        assert self.test_shape.solid is not None
         assert test_shape.solid is not None
```
Comment (Shimwell) [inline tests/test_parametric_components/test_BlanketCutterParallels.py:50]:
```suggestion
        assert self.test_shape.solid is not None
```

**Pull request merged**